framework/web/User.php

```diff
@@ -700,18 +701,21 @@ protected function renewAuthStatus()
             $identity = $class::findIdentity($id);
         }
 
-        if ($identity !== null) {
-            $authKey = $session->get($this->authKeyParam);
-            if ($authKey !== null && !$identity->validateAuthKey($authKey)) {
-                $identity = null;
-                $ip = Yii::$app->getRequest()->getUserIP();
-                Yii::warning("Invalid session auth key attempted for user '$id' from $ip: $authKey", __METHOD__);
-            }
+        if ($identity === null) {
+            $this->switchIdentity(null);
+            return;
+        }
```

Comment:
@bizley I think you were right: We can do this here.

If we agree that `$identity` is the main source of truth, then that means: If there is no identity anymore (`findIdentity() === null`) then the user is no longer authenticated. We can stop any auto login attempts and don't need to check the auth key etc.